governance-and-automation/security-compliance-checks/security-compliance-checks.md

```diff
@@ -32,15 +38,17 @@ After clicking View All, the Security and Compliance checks dashboard will open,
 
 * **Workspaces with not enough owners (4)** – shows the number of workspaces that don't have enough owners. You will be able to investigate further and assign additional owners if needed. Keep an eye on workspaces with only 1 owner because **if that one owner leaves, a workspace is left without an owner**. 
 
-* **Orphaned users (5)** - shows the number of orphaned users and takes you to the Orphaned Users report, which helps **detect and remove blocked/disabled or deleted users** from your Microsoft 365 resources. 
+* **Orphaned users (5)** - shows the number of orphaned users and takes you to the Orphaned Users report, which helps **detect and remove blocked or deleted users** from your Microsoft 365 workspaces. 
```

Comment:
možda ubaciti hyperlink pod Orphaned Users report, na kojem ljudi mogu kliknuti i vidjeti što točno orphaned users report radi https://docs.syskit.com/point/reporting/cleanup-and-health-reports#orphaned-users, nije nužno u ovoj verziji, ali možda za razmisliti za ubuduće, nekak ljude natjerati da pročitaju te članke o reportima i što točno rade.

Reply:
ovo sad vodi na report koji nije isti kao i Orphaned Users report u report centru, pa za sad neću linkati
ali noted za neku sljedeću iteraciju